--- OSMF.orig/framework/OSMF/org/osmf/net/httpstreaming/f4f/AdobeFragmentRunTable.as	2012-03-21 10:45:00.000000000 +0100
+++ OSMF/framework/OSMF/org/osmf/net/httpstreaming/f4f/AdobeFragmentRunTable.as	2014-06-19 17:31:49.716598212 +0200
@@ -386,7 +386,7 @@
 		
 		private function calculateFragmentId(fdp:FragmentDurationPair, time:Number):uint
 		{
-			if (fdp.duration <= 0)
+			if (fdp.duration <= 0 || time < fdp.durationAccrued)
 			{
 				return fdp.firstFragment;
 			}
