--- OSMF.orig/framework/OSMF/org/osmf/elements/ImageLoader.as	2012-03-21 10:45:08.000000000 +0100
+++ OSMF/framework/OSMF/org/osmf/elements/ImageLoader.as	2014-06-19 17:29:04.301915563 +0200
@@ -89,6 +89,12 @@
 				urlResource.url != null)
 			{
 				var url:URL = new URL(urlResource.url);
+				
+				// Allow Vortex resource thumbnail URLs
+				if (url.getParamValue("vrtx") == "thumbnail") {
+					return true;
+				}
+				
 				return (url.path.search(/\.gif$|\.jpg$|\.png$/i) != -1);
 			}	
 			return false;
